vizro-core/src/vizro/models/_navigation/_navigation_utils.py

```diff
@@ -0,0 +1,21 @@
+import itertools
+
+from vizro.managers import model_manager
+
+
+def _validate_pages(pages):
+    """Validator for reuse in `Navigation` and `Accordion` to validate pages."""
```

Comment:
If we're actually writing a docstring here, shall we quickly say what this validates?

Reply:
Changed to: `Reusable validator in `Navigation` and `Accordion` to check if provided Page IDs exist as registered pages.`